kube/services/jobs/data-ingestion-job.yaml

```diff
@@ -0,0 +1,156 @@
+          secret:
+            secretName: "fence-secret"
+        - name: json-secret-volume
+          secret:
+            secretName: "fence-json-secret"
+        - name: creds-volume
+          secret:
+            secretName: "fence-creds"
+        - name: config-helper
+          configMap:
+            name: config-helper
+        - name: config-volume
+          secret:
+            secretName: "fence-config"
+        - name: fence-google-app-creds-secret-volume
+          secret:
+            secretName: "fence-google-app-creds-secret"
+        - name: fence-google-storage-creds-secret-volume
+          secret:
+            secretName: "fence-google-storage-creds-secret"
+      containers:
+        - name: data-ingestion-pipeline
+          GEN3_DATA-INGESTION-PIPELINE_IMAGE
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 80
+          env:
+            - name: CREDS_JSON
+              valueFrom:
+                secretKeyRef:
+                  name: data-ingestion-job-secret
+                  key: config.json
+          volumeMounts:
+            - name: shared-data
+              mountPath: /this-directory-is-shared-with-fence
+            - name: phs-id-list
+              readOnly: true
+              mountPath: /phs-id-list
+            - name: data-requiring-manual-review
+              readOnly: true
+              mountPath: /data-requiring-manual-review
+          command: ["/bin/bash" ]
+          args:
+            - "-c"
+            - |
+              ./run.sh
+
+              if [ -f "scripts/joindure/output/google-groups.sh" ]; then
+                  echo "google-groups.sh contains:"
+                  cat scripts/joindure/output/google-groups.sh
+              else
+                echo "Error: google groups generation script is not present."
+              fi
+
+              cp -R scripts/joindure/output/. /this-directory-is-shared-with-fence/.
+        - name: fence
+          GEN3_FENCE_IMAGE
+          imagePullPolicy: Always
+          env:
+            - name: PYTHONPATH
+              value: /var/www/fence
+          volumeMounts:
+            - name: shared-data
+              mountPath: /data-ingestion-pipeline-output
+            - name: "config-volume"
+              readOnly: true
+              mountPath: "/var/www/fence/fence-config.yaml"
+              subPath: fence-config.yaml
+            - name: "fence-google-app-creds-secret-volume"
+              readOnly: true
+              mountPath: "/var/www/fence/fence_google_app_creds_secret.json"
+              subPath: fence_google_app_creds_secret.json
+            - name: "fence-google-storage-creds-secret-volume"
+              readOnly: true
+              mountPath: "/var/www/fence/fence_google_storage_creds_secret.json"
+              subPath: fence_google_storage_creds_secret.json
+          command: ["/bin/bash" ]
+          args:
+            - "-c"
+            - |
+              while true; do
```

Comment:
Are you sure you don't want to move this script into the docker image, and just call `run.sh` - something like that?

Reply:
dev/test on the script is a lot easier if you can just run it directly ...

Reply:
done ! moved the script to the other repo https://github.com/uc-cdis/dataSTAGE-data-ingestion/blob/215e95bc1e9b898cba52fe15515b77fb1dedf003/scripts/fence-image-commands.sh